packages/crusher-app/src/store/selectors/getCurrentProject.ts

```diff
@@ -6,5 +6,5 @@ import { projectsAtom } from "../atoms/global/project";
 export const currentProjectSelector = atom((get) => {
 	const { selectedProjectId } = get(appStateAtom);
 	const projects = get(projectsAtom);
-	return projects.filter(({ id }) => selectedProjectId === id)[0];
+	return projects?.filter(({ id }) => selectedProjectId === id)[0];
```

Comment:
Wrong optional chaining here.Two issues
1.) This is duct tape solution. We're now expecting project to have null value.
2.) Inferred type will get change throughout this state.